client/jetpack-connect/index.js

```diff
@@ -34,33 +35,21 @@ export default function () {
 		'scan',
 		'realtimebackup',
 		'antispam',
+		'jetpack_search',
+		'wpcom_search',
+		...OFFER_RESET_FLOW_TYPES,
```

Comment:
Supporting Offer Reset product slugs. Without this, a URL like `wordpress.com/jetpack/connect/jetpack_security_daily` wouldn't work.